src/js/common/schemaform/FormSaved.jsx

```diff
@@ -31,22 +32,33 @@ class FormSaved extends React.Component {
     }
   }
 
+  verifyUser = () => {
+    handleVerify(this.props.user.login.verifyUrl);
+  }
+
   render() {
     const { profile } = this.props.user;
     const lastSavedDate = this.props.lastSavedDate;
     const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
+    const verifiedAccountType = 3;// verified ID.me accounts are type 3
+    const notVerified = profile.accountType !== verifiedAccountType;
     const { success } = this.props.route.formConfig.savedFormMessages || {};
 
     return (
       <div>
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <strong>Your application has been saved!</strong><br/>
-            {!!lastSavedDate && <p>Last saved on {moment(lastSavedDate).format('M/D/YYYY [at] h:mma')}.</p>}
+            <strong>Your application has been saved.</strong><br/>
+            {!!lastSavedDate && <p>Last saved on {moment(lastSavedDate).format('M/D/YYYY [at] h:mm a')}</p>}
             {success}
-            If you're on a public computer, please sign out before you leave to ensure your data is secure.
+            If you're logged in through a public computer, please sign out of your account before you log off to keep your information secure.
           </div>
         </div>
+        {notVerified && <div className="usa-alert usa-alert-warning">
+          <div className="usa-alert-body">
+            We want to keep your information safe with the highest level of security. Please <a href="#" onClick={this.verifyUser}>verify your identity</a>.
```

Comment:
I've been trying to make these buttons, since they're actions and not navigation. There's a class called `va-button-link` that styles buttons like links, so it would be: `<button className="va-button-link" onClick={this.verifyUser}>verify your identity</button>`.